Mathlib/Tactic/Linter/Lint.lean

```diff
@@ -158,4 +158,66 @@ initialize addLinter missingEndLinter
 
 end MissingEnd
 
+/-!
```

Comment:
This is starting a bikeshed, but: intuitively, I would rather add this to `Linters/Style`. (This linter is purely about coding style, and not at all relevant to correctness.) What do you think?

I agree this starts a broader conversation how to structure the `Linters` directory: would we rather have separate files for individual linters, or are we fine grouping linters if each is reasonably short? (In any case, having an overview of all linters in some doc-string would be nice --- but that's definitely out of scope for this PR.)

Reply:
I agree that this is more "stylistic" than "correctness", but the other linters also do not really check correctness.

Anyway, I am happy if this linter goes somewhere else, but, I also agree that this is a discussion that should happen elsewhere!

Reply:
So I'll start a zulip discussion/propose a re-organisation in a PR? Sounds good to me.